website/templates/public/pages/help/addons.mako

```diff
@@ -67,3 +67,43 @@
 <p>Next, authenticate with Dataverse by entering your Dataverse username and password and clicking "Submit". You can then choose the Dataverse study you would like to add to your OSF project. Click "Submit" to save your settings.</p>
 <p>Contributors to your project will have access to both released and draft versions of your study, but only the most recent release will be made public alongside your OSF project.</p>
 
+<span id="box" class="anchor"></span>
+<p class="gs-header">Box</p>
+<p><strong><a href="http://box.com/">Box</a> is an online file storage program.</strong>After linking your OSF project to Box, you can add files to your Box folder and those files can be accessed via the OSF. Likewise, files added to the Box folder in your OSF account will update your Box account with that file.</p>
+<p>To link a Box folder to a project/component, first visit the project you want to add a Box folder to. Then go to "Settings" in the grey navigation bar. Check "Box" under "Select Add-ons" to enable the add-on. Read, click "OK" on the pop-up, then submit.</p>
+<p>Next, you will be asked to authenticate with Box by clicking the "Authorize" button. Once you have said "OK" you can choose the folder you would like to add to your OSF project.</p>
+
+
+<span id="drive" class="anchor"></span>
+<p class="gs-header">Google Drive </p>
+<p><strong><a href="http://drive.google.com/">Google Drive</a> is a popular file storage and collaboration tool. </strong>After linking your OSF project to Google Drive, you will be able to access and import your Google Drive folders. Likewise, files added to the Google Drive folder in your OSF account will update the folder in Google Drive. </p>
+<p>To enable, navigate to settings in your project, then navigate to "Select Add-Ons."  Select Google Drive, and click submit. You will be asked to give permission for the OSF to interact with Google Drive. Click "Accept" to create your access token for your entire user account.</p>
+<p>Then navigate to "Configure Add-Ons" to import the access token that you created in the previous step. Select "Import Access Token" and then select the folders you want to sync.</p>
+<p>Now you will be able to find your Google Drive folder in your "Files" section. </p>
+
+
+<h3> Citation Managers </h3>
+<div class="gs-video embed-responsive embed-responsive-16by9">
+    <div class="embed-responsive-item youtube-loader" id="2RVJS7nYX8k"></div>
+</div>
+
+
+<span id="zotero" class="anchor"></span>
```

Comment:
The ordering with the side navbar is different; Mendeley goes before Zotero. I will fix this when I merge.